diff --git a/GXRuntime/graphics/aurora/lib/gfx/gxcore_draw.cpp b/GXRuntime/graphics/aurora/lib/gfx/gxcore_draw.cpp
index a6203f1..ceaa44c 100644
--- a/GXRuntime/graphics/aurora/lib/gfx/gxcore_draw.cpp
+++ b/GXRuntime/graphics/aurora/lib/gfx/gxcore_draw.cpp
@@ -519,8 +519,8 @@ wgpu::RenderPipeline create_pipeline(const PipelineConfig& config) {
     });
   }
   const wgpu::VertexBufferLayout vertexLayout{
-      .arrayStride = gxc::kVertexStrideBytes,
       .stepMode = wgpu::VertexStepMode::Vertex,
+      .arrayStride = gxc::kVertexStrideBytes,
       .attributeCount = attributes.size(),
       .attributes = attributes.data(),
   };
